src/lib.rs

```diff
@@ -523,8 +540,27 @@ impl<'a, Writer: Write> Printer<'a, Writer> {
         Ok(())
     }
 
+    fn reorder_buf_to_le(&self, buf: &mut Vec<u8>) {
```

Comment:
I'd prefer a non-abbreviated name
```suggestion
    fn reorder_buffer_to_little_endian(&self, buf: &mut Vec<u8>) {
```